VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -111,6 +120,16 @@ public void onPageLoaded(List<T> newItems, boolean hasMore) {
                     hasMoreItems = hasMore;
                     loading = false;
                 }
+
+                /**
+                 * Return whether this callback has been abandoned because of the controller being reset.
+                 *
+                 * @return <code>true</code> if the callback has been reset otherwise <code>false</code>
```

Comment:
The return condition should also refer to "abandoned" instead of "reset" I think.